projects/packages/classic-theme-helper/webpack.config.js

```diff
@@ -0,0 +1,94 @@
+const path = require( 'path' );
+const jetpackWebpackConfig = require( '@automattic/jetpack-webpack-config/webpack' );
+const glob = require( 'glob' );
+
+const sharedWebpackConfig = {
+	mode: jetpackWebpackConfig.mode,
+	devtool: jetpackWebpackConfig.devtool,
+	output: {
+		...jetpackWebpackConfig.output,
+		path: path.resolve( './build' ),
+	},
+	optimization: {
+		...jetpackWebpackConfig.optimization,
+	},
+	resolve: {
+		...jetpackWebpackConfig.resolve,
+		modules: [ 'node_modules' ],
+		alias: {
+			...jetpackWebpackConfig.resolve.alias,
+			fs: false,
+		},
+	},
+	node: {},
+	externals: {
+		...jetpackWebpackConfig.externals,
+		jetpackConfig: JSON.stringify( {
+			consumer_slug: 'jetpack-classic-theme-helper',
+		} ),
+	},
+	module: {
+		strictExportPresence: true,
+		rules: [
+			// Transpile JavaScript
+			jetpackWebpackConfig.TranspileRule( {
+				exclude: /node_modules\//,
+			} ),
+
+			// Handle CSS.
+			jetpackWebpackConfig.CssRule( {
+				extensions: [ 'css', 'sass', 'scss' ],
+				extraLoaders: [
+					{
+						loader: 'postcss-loader',
+						options: {
+							postcssOptions: { plugins: [ require( 'autoprefixer' ) ] },
+						},
+					},
+					{
+						loader: 'sass-loader',
+						options: {
+							sassOptions: {
+								// The minifier will minify if necessary.
+								outputStyle: 'expanded',
+							},
+						},
+					},
+				],
+			} ),
+
+			// Leave fonts and images in place.
+			{
+				test: /\.(eot|ttf|woff|png|svg)$/i,
+				type: 'asset/resource',
+				generator: {
+					emit: false,
+					filename: '[file]',
+				},
+			},
+		],
+	},
+	plugins: [
+		...jetpackWebpackConfig.StandardPlugins( {
+			DependencyExtractionPlugin: false,
+			I18nLoaderPlugin: false,
+			I18nCheckPlugin: false,
+			MiniCssWithRtlPlugin: false,
+			WebpackRtlPlugin: false,
+		} ),
+	],
```

Comment:
This also could be simplified. It would be good to generate an `.rtl.css` file too for production builds, as the previous build (in `jetpack/_inc/build/theme-tools/responsive-videos`) had that, as well as an `asset.php` file. The standard configuration works here too for that (though possibly it could be refined further - I'm not too familiar with the 118n plugins for example and whether we should be setting them to false).

```suggestion
	plugins: [ ...jetpackWebpackConfig.StandardPlugins() ],
```